app/controllers/games.js

```diff
@@ -30,3 +40,17 @@ exports.list = (req, res, next) => {
     next(err);
   });
 };
+
+exports.createMatch = (req, res, next) => {
```

Comment:
Maybe we should consider moving these to a `matchsController`, so we have one controller for each model.